src/ViewTrait.php

```diff
@@ -514,6 +531,7 @@ public function clear(): void
     {
         $this->viewFiles = [];
         $this->state->clear();
+        $this->localeState = new LocaleState();
```

Comment:
May be add method `LocaleState::clear()` and use him here?
Only suggestion for consistent with `ViewState` and `WebViewState`.

Reply:
This is only for internal use, I don’t see much point in creating an additional method when you can create a new instance.